Gems/Prefab/PrefabBuilder/PrefabGroup/PrefabGroupBehavior.cpp

```diff
@@ -376,7 +376,28 @@ namespace AZ::SceneAPI::Behaviors
         ManifestAction action,
         [[maybe_unused]] RequestingApplication requester)
     {
-        if (action != Events::AssetImportRequest::ConstructDefault)
+        if (action == Events::AssetImportRequest::Update)
+        {
+            // clear any constructed procedural prefabs
+            AZStd::vector<const SceneData::PrefabGroup*> emptyPrefabGroupCollection;
```

Comment:
nit: to avoid memory reallocations when adding elements to this vector it could do `emptyPrefabGroupCollection.reserve(manifest.GetEntryCount());`.

Reply:
this logic has been removed, please re-evaluate the PR